src/cli/config.rs

```diff
@@ -0,0 +1,62 @@
+use crate::{api_client::Client, types::StateId};
+use clap::{Args, Parser, Subcommand};
+use std::fmt;
+
+#[derive(Debug, Parser)]
+#[clap(version, about = "Beacon API client")]
+pub struct CliConfig {
+    #[clap(short, long)]
```

Comment:
```suggestion
    #[clap(long)]
```

imo `-e` is a little confusing and I'd rather only have the long form